app/models/main.py

```diff
@@ -1832,6 +1848,307 @@ def serialize(self):
         }
 
 
+class ProcessOutcome(db.Model):
+    __tablename__ = "process_outcomes"
+
+    RESULT_CHOICES = (
+        "awarded",
+        "cancelled",
+        "none-suitable",
+    )
+
+    id = db.Column(db.Integer, primary_key=True)
+    external_id = db.Column(db.BigInteger, nullable=False, default=random_positive_external_id, unique=True)
+    completed_at = db.Column(db.DateTime, nullable=True)
+
+    start_date = db.Column(db.Date, nullable=True)
+    end_date = db.Column(db.Date, nullable=True)
+    awarding_organisation_name = db.Column(db.String, nullable=True)
+    # should be enough scale to represent up to £1b - 1p
+    award_value = db.Column(db.Numeric(11, 2), nullable=True)
+
+    result = db.Column(db.String, nullable=False)
+
+    direct_award_project_id = db.Column(
```

Comment:
Same as with `brief` and `brief_response` this can be accessed through the direct award search attribute?

Reply:
Access is not the role of this field. The purpose is to provide a column to add a unique constraint for. Same goes for `brief_id`.